templates/index.html

```diff
@@ -1,55 +1,189 @@
+
+<body>
+
+  <!-- Navigation -->
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
+    <div class="container">
+      <a class="navbar-brand" href="#">TAG</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
+        aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarResponsive">
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Home
+              <span class="sr-only">(current)</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Services</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <!-- Page Content -->
+  <!-- Labels -->
+  <div class="container">
+
+    <div class="row">
+
+      <div class="col-lg-2">
+        <div class="my-2 row bg-light rounded-lg">
+          <div class="col-lg-12 mt-2 mb-3">
+            <h2 class="my-2">Labels</h2>
+
+            <div id="labels" class="list-group">
+              <div class="list-group-item py-2 px-3 label label-selected" value="lorem"
+                style="background-color: lightcoral;">Label 1</div>
+              <div class="list-group-item py-2 px-3 label" value="dolor"
+                style="background-color: yellow;">Label 2</div>
+              <div class="list-group-item py-2 px-3 label" value="rem"
+                style="background-color: pink;">Label 3</div>
+              <div class="list-group-item py-2 px-3 label" value="sit"
+                style="background-color: greenyellow;">Label 4</div>
+            </div>
+          </div>
+        </div>
+        <div class="my-3 row">
+          <div class="col-lg-3"></div>
+          <button id="export" type="button" class="btn btn-primary col-lg-6">Export</button>
+          <div class="col-lg-3"></div>
+        </div>
+        <div class="my-2 row bg-light rounded-lg">
+          <div class="col-lg-12 mt-2 mb-3">
+            <h2 class="my-2">Colors</h2>
+            <p>Please select a color</p>
+            <div class="highlight-color color-selected" style="background-color: red" id="red">
+              <p></p>
+            </div>
+            <div class="highlight-color" style="background-color: orange" id="orange">
             </div>
+            <div class="highlight-color" style="background-color: yellow" id="yellow">
+            </div>
+            <div class="highlight-color" style="background-color: green" id="green">
+            </div>
+            <div class="highlight-color" style="background-color: blue" id="blue">
```

Comment:
I see the colors are defined but they are not showing up in my browser

Reply:
Which browser are you using?